data/opam_docs/2.1/03_faq.md

```diff
@@ -0,0 +1,446 @@
+---
+slug: faq
+title: FAQ
+description: |
+  Answers to frequently asked questions about Opam.
+---
+
+# opam FAQ
+
+> This FAQ is for general questions about opam and its usage. You may also be
+> interested in the more advanced [Tricks](tricks) for specific use-cases
+> or advanced users.
+
+#### 🐫  What is opam for?
+
+Easily installing, upgrading and managing your OCaml compiler(s), tools and
+libraries. It's made of a tool, the
+[opam package manager](https://github.com/ocaml/opam), and a
+community-maintained
+[package repository](https://github.com/ocaml/opam-repository).
+
+Note that the tool itself is not specific to OCaml at its core, and could be
+used for different systems using specific repositories (e.g. for the
+[Coq theorem prover](http://coq.io/opam/))
+
+---
+
+#### 🐫  How to get, install and upgrade opam?
+
+See the [install guide](install).
+
+If upgrading, you can bootstrap using `opam install opam-devel`, and follow the
+instructions.
+
+---
+
+#### 🐫  Where is the manual?
+
+opam has git-like, hierarchical
+manpages. Try `opam --help` for
+a starting point.
+
+Or get started from the [Usage](usage) guide.
+
+If you want to know more about opam packages, see the [Packaging Howto](packaging).
+
+The full reference on the internals and file formats is in the [Manual](manual).
+
+You may also want to browse the [library APIs](api/).
+
+---
+
+#### 🐫  What changes does opam do to my filesystem?
+
+opam is designed to be run strictly as user (non-root), and except for the
+explicit options provided during `opam init`, opam only writes within `~/.opam`
+(and `/tmp`). This directory — the default "opam root" — contains configuration,
+various internal data, a cache of downloaded archives, and your OCaml
+installations.
+
+If you choose to create "local switches", the installation prefix will be put in
+the specified directory with `/_opam/` appended. Nothing else will be changed.
+
+Please note, however, that programs you install using opam won't themselves be
+bound by any restrictions.
+
+On Linux, and since opam 2.0.0~rc2, package instructions (build, install,
+remove) are also run in a sandbox and guaranteed not to affect your system.
+
+---
+
+#### 🐫  Why does opam require ``bwrap``?
```

Comment:
```suggestion
#### 🐫  Why Does opam Require ``bwrap``?
```

Reply:
Is there a reason this one is wrapped in double ` rather than single as monospace is usually done?